docs/upgrade.html

```diff
@@ -19,6 +19,13 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_4_1_0" href="#upgrade_4_1_0">Upgrading to 4.1.0 from any version 0.8.x through 3.9.x</a></h4>
+    <h5><a id="upgrade_410_notable" href="#upgrade_410_notable">Notable changes in 4.1.0</a></h5>
+    <ul>
+        <li>The <code>control.plane.listener.name</code> config was removed use <code>controller.listener.names</code> instead.
```

Comment:
Changes like this need to be done in a major release (4.0).

Reply:
That said, I think I would not list each of these zk removal related changes. Instead, we should just have a single message about removal of ZK support. And then we should have a separate page for migrating from zk to kraft that includes all the relevant details, including this one.

Reply:
file https://issues.apache.org/jira/browse/KAFKA-18364